### ISSUE TITLE
Simple README update

### ISSUE THREAD

**Issue by arosen93** [glotzerlab/signac#912]
The quickstart in the README states:
```
$ mkdir my_project
$ cd my_project
$ signac init MyProject
```

However, I think it should be
```
$ mkdir my_project
$ cd my_project
$ signac init
```

**Comment by bdice** [comment]
Good call. We just released signac 2.0.0 which has some breaking changes. If you notice anything else out of place, please let us know!

**Comment by bdice** [comment]
Thanks for filing this @arosen93. It is fixed in #913.

**Comment by arosen93** [comment]
@bdice --- will do!!I'm excited to give signac a spin. Congrats on the release!
